Lima v2.0 fails to install properly

The action fails to install Lima v2.0.0 and above due permissions error related to the missing `libexec` folder:

```
Installing Lima version v2.0.1
  x ./
  x ./bin/
  x ./libexec/: Can't create 'libexec'
  x ./share/
  x ./share/man/
  x ./share/lima/
  x ./share/doc/
  x ./share/doc/lima/
  x ./share/doc/lima/LICENSE
  x ./share/doc/lima/MAINTAINERS.md
  x ./share/doc/lima/README.md
  x ./share/doc/lima/ROADMAP.md
  x ./share/doc/lima/VERSION
  x ./share/doc/lima/templates
  x ./share/doc/lima/SECURITY.md
  x ./share/lima/lima-guestagent.Linux-x86_64.gz
  x ./share/lima/templates/
  x ./share/lima/templates/ubuntu-25.10.yaml
  x ./share/lima/templates/podman-rootful.yaml
  x ./share/lima/templates/experimental/
  x ./share/lima/templates/centos-stream-10.yaml
  x ./share/lima/templates/fedora-43.yaml
  x ./share/lima/templates/buildkit.yaml
  x ./share/lima/templates/oraclelinux-9.yaml
  x ./share/lima/templates/oraclelinux-8.yaml
  x ./share/lima/templates/fedora-42.yaml
  x ./share/lima/templates/debian.yaml
  x ./share/lima/templates/ubuntu-20.04.yaml
  x ./share/lima/templates/ubuntu-24.10.yaml
  x ./share/lima/templates/rocky.yaml
  x ./share/lima/templates/alpine.yaml
  x ./share/lima/templates/k8s.yaml
  x ./share/lima/templates/almalinux-kitten.yaml
  x ./share/lima/templates/apptainer.yaml
  x ./share/lima/templates/opensuse-leap-15.yaml
  x ./share/lima/templates/ubuntu-22.04.yaml
  x ./share/lima/templates/opensuse-leap.yaml
  x ./share/lima/templates/docker-rootful.yaml
  x ./share/lima/templates/almalinux.yaml
  x ./share/lima/templates/faasd.yaml
  x ./share/lima/templates/ubuntu.yaml
  x ./share/lima/templates/fedora.yaml
  x ./share/lima/templates/README.md
  x ./share/lima/templates/almalinux-8.yaml
  x ./share/lima/templates/k0s.yaml
  x ./share/lima/templates/almalinux-10.yaml
  x ./share/lima/templates/_images/
  x ./share/lima/templates/_default/
  x ./share/lima/templates/archlinux.yaml
  x ./share/lima/templates/debian-13.yaml
  x ./share/lima/templates/rocky-8.yaml
  x ./share/lima/templates/rocky-9.yaml
  x ./share/lima/templates/opensuse-leap-16.yaml
  x ./share/lima/templates/opensuse.yaml
  x ./share/lima/templates/debian-12.yaml
  x ./share/lima/templates/alpine-iso.yaml
  x ./share/lima/templates/oraclelinux-10.yaml
  x ./share/lima/templates/podman.yaml
  x ./share/lima/templates/almalinux-9.yaml
  x ./share/lima/templates/docker.yaml
  x ./share/lima/templates/centos-stream.yaml
  x ./share/lima/templates/k3s.yaml
  x ./share/lima/templates/almalinux-kitten-10.yaml
  x ./share/lima/templates/rocky-10.yaml
  x ./share/lima/templates/centos-stream-9.yaml
  x ./share/lima/templates/ubuntu-25.04.yaml
  x ./share/lima/templates/apptainer-rootful.yaml
  x ./share/lima/templates/fedora-41.yaml
  x ./share/lima/templates/linuxbrew.yaml
  x ./share/lima/templates/oraclelinux.yaml
  x ./share/lima/templates/ubuntu-lts.yaml
  x ./share/lima/templates/default.yaml
  x ./share/lima/templates/ubuntu-24.04.yaml
  x ./share/lima/templates/_default/mounts.yaml
  x ./share/lima/templates/_images/ubuntu-25.10.yaml
  x ./share/lima/templates/_images/centos-stream-10.yaml
  x ./share/lima/templates/_images/fedora-43.yaml
  x ./share/lima/templates/_images/oraclelinux-9.yaml
  x ./share/lima/templates/_images/oraclelinux-8.yaml
  x ./share/lima/templates/_images/fedora-42.yaml
  x ./share/lima/templates/_images/ubuntu-20.04.yaml
  x ./share/lima/templates/_images/ubuntu-24.10.yaml
  x ./share/lima/templates/_images/alpine.yaml
  x ./share/lima/templates/_images/opensuse-leap-15.yaml
  x ./share/lima/templates/_images/ubuntu-22.04.yaml
  x ./share/lima/templates/_images/opensuse-leap.yaml
  x ./share/lima/templates/_images/debian-11.yaml
  x ./share/lima/templates/_images/ubuntu.yaml
  x ./share/lima/templates/_images/fedora.yaml
  x ./share/lima/templates/_images/almalinux-8.yaml
  x ./share/lima/templates/_images/almalinux-10.yaml
  x ./share/lima/templates/_images/archlinux.yaml
  x ./share/lima/templates/_images/debian-13.yaml
  x ./share/lima/templates/_images/rocky-8.yaml
  x ./share/lima/templates/_images/rocky-9.yaml
  x ./share/lima/templates/_images/opensuse-leap-16.yaml
  x ./share/lima/templates/_images/debian-12.yaml
  x ./share/lima/templates/_images/alpine-iso.yaml
  x ./share/lima/templates/_images/oraclelinux-10.yaml
  x ./share/lima/templates/_images/almalinux-9.yaml
  x ./share/lima/templates/_images/almalinux-kitten-10.yaml
  x ./share/lima/templates/_images/rocky-10.yaml
  x ./share/lima/templates/_images/centos-stream-9.yaml
  x ./share/lima/templates/_images/ubuntu-25.04.yaml
  x ./share/lima/templates/_images/fedora-41.yaml
  x ./share/lima/templates/_images/ubuntu-lts.yaml
  x ./share/lima/templates/_images/ubuntu-24.04.yaml
  x ./share/lima/templates/experimental/gentoo.yaml
  x ./share/lima/templates/experimental/opensuse-tumbleweed.yaml
  x ./share/lima/templates/experimental/vnc.yaml
  x ./share/lima/templates/experimental/wsl2.yaml
  x ./share/lima/templates/experimental/ubuntu-26.04.yaml
  x ./share/lima/templates/experimental/u7s.yaml
  x ./share/lima/templates/experimental/alsa.yaml
  x ./share/lima/templates/experimental/ubuntu-next.yaml
  x ./share/lima/templates/experimental/debian-sid.yaml
  x ./share/lima/templates/experimental/rke2.yaml
  x ./share/man/man1/
  x ./libexec/lima/: Failed to create dir 'libexec'
  x ./libexec/lima/limactl-mcp: Failed to create dir 'libexec'
  x ./bin/docker.lima
  x ./bin/lima
  x ./bin/podman.lima
  x ./bin/nerdctl.lima
  x ./bin/kubectl.lima
  x ./bin/limactl
  x ./bin/apptainer.lima
  tar: Error exit delayed from previous errors.
  Error: Process completed with exit code 1.
```

--- a/action.yml
+++ b/action.yml
@@ -65,6 +65,12 @@ runs:
         fi
 
         echo "::group::Installing Lima version $LIMA_VERSION"
+        
+        if [[ $LIMA_VERSION == v2.* ]]; then
+            sudo mkdir /usr/local/libexec
+            sudo chown $UID:$GID /usr/local/libexec
+        fi
+
         curl -fsSL "https://github.com/lima-vm/lima/releases/download/${LIMA_VERSION}/lima-${LIMA_VERSION:1}-$(uname -s)-$(uname -m).tar.gz" | tar Cxzvm /usr/local
         echo "::endgroup::"
       shell: bash

